src/slider/index.ts

```diff
@@ -0,0 +1,140 @@
+import { tag, utils, Tag } from '@storefront/core';
+
+@tag('gb-slider', require('./index.html'), require('./index.css'))
+class Slider {
+
+  props: Slider.Props = {
+    defaultLower: 0,
```

Comment:
can this be `min` and `max`?

Reply:
oh, or is this where the handles start?
can we use a `prop` called `value` that's set to a two-element array?
so the `lower` and `upper` values? and then we can default to `value = [0, 100]`